sklearn/model_selection/_split.py

```diff
@@ -758,6 +762,7 @@ class TimeSeriesSplit(_BaseKFold):
 
     max_train_size : int, optional
         Maximum size for a single training set.
+        ..versionadded:: v0.19
```

Comment:
You might need a whitespace before `versionadded`

Reply:
Ok adding it in.